app/common/lib/windowsUtil.js

```diff
@@ -3,11 +3,47 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const Immutable = require('immutable')
+const computedStyles = require('../constants/computedStyles')
+const windowState = require('../state/windowState')
+const appActions = require('../../../js/actions/appActions')
+const {getCurrentWindowId} = require('../../renderer/currentWindow')
 
-module.exports.getPinnedSiteProps = site => {
+const getPinnedSiteProps = site => {
   return Immutable.fromJS({
     location: site.get('location'),
     order: site.get('order'),
     partitionNumber: site.get('partitionNumber') || 0
   })
 }
+
+const getComputedStyleData = (root) => {
+  const result = {}
+
+  Object.keys(computedStyles).forEach(group => {
+    computedStyles[group].forEach(prop => {
+      const value = `--${group}-${prop}`
+      result[value] = root.getPropertyValue(value).trim()
+    })
+  })
+
+  return result
+}
+
+const getComputedProperty = (state, property) => {
```

Comment:
I think this should be replaced by a `getComputedStyles` method in `windowState`. We don't want anything outside of windowState to be tightly coupled to the storage path